front-end/src/app/shared/services/validate.service.spec.ts

```diff
@@ -83,6 +83,7 @@ describe('ValidateService', () => {
     let result: ValidationErrors | null = validator(
       service.formValidatorForm.get('L6a_cash_on_hand_jan_1_ytd') as FormControl
     );
+    expect(result).not.toEqual(null);
```

Comment:
I saw this removed in a pr from @mjtravers.  Is it supposed to be back?

Reply:
It was a check that was failing because it didn't like using `toBe(null)`

It could be removed, it could be changed, I'd fixed it separately and I just kept it in.

Reply:
It can be in or out. The test is not consequential which is why I removed it.